views/partials/banner_breach.hbs

```diff
@@ -13,6 +13,6 @@
     <span class="breach-info"><span class="medium">Compromised Data</span> {{breachDataClasses breach.meta.DataClasses }}</span>
     <span class="breach-info"><span class="medium">Compromised Accounts</span> {{localeString breach.meta.PwnCount }}</span>
     <span class="breach-info">{{{ breach.meta.Description }}}</span>
-    <span class="source-info">Breach data provided by<span class="demi"> Have I Been Pwned?</span></span>
+    <span class="source-info">Breach data provided by<a class="demi" href="https://www.haveibeenpwned.com" alt="Have I Been Pwned"> Have I Been Pwned</a></span>
```

Comment:
`<a>` tags don't support `alt=""`. You could use `aria-label` if you want to set something special for a11y, but since the text and the `alt` are the same, you can probably just remove it here (and in the diff below).

Reply:
![](https://media1.tenor.com/images/5e4cbcc82e6f77134b69a453645ed136/tenor.gif?itemid=4424463)